docs/README.md

```diff
@@ -440,30 +324,32 @@ The `validate` action is available as [`@demos/validate.js`](https://github.com/
 composition as [`@demos/try.js`](https://github.com/ibm-functions/shell/blob/master/app/demos/try.js) for your convenience.
 
 ```bash
+# enter in Shell
 # create validate action
-$ fsh action create validate @demos/validate.js
+> action create validate @demos/validate.js
 
 # create app
-$ fsh app create try @demos/try.js
+> app create try @demos/try.js
 
 # invoke app with valid parameter
-$ fsh app invoke try -p str aGVsbG8gdHJ5IQ==
+> app invoke try -p str aGVsbG8gdHJ5IQ==
 {
   ok: true
 }
 
 # and now for the failing case
-$ fsh app invoke try -p str bogus
+> app invoke try -p str bogus
```

Comment:
@tardieu had a pr where you can use fsh inside the shell and it’s ignored. It is convenient because then you can past the same command headless or not.